kubearmor_receiver/go.mod

```diff
@@ -1,57 +1,66 @@
 module github.com/kubearmor/OTel-receiver/kubearmor_receiver
 
-go 1.20
+go 1.21
```

Comment:
Can we use `go 1.22` with `toolchain go1.22.1`
Go 1.21 had some vulnerabilities. You can refer to https://github.com/kubearmor/kubearmor-client/pull/414 for the same.